igor.dzetcyna/1/lesson_3/lesson3_class_PrimeNumber.rb

```diff
@@ -0,0 +1,30 @@
+class PrimeNumber
+
+  def initialize(max_value)
+    @value = max_value
+    @prime_arr = []
+  end
+
+  def find_prime_numbers
+    (2..@value).each do |n|
+      if (1..n).find_all { |el| n % el == 0 } == [1, n]
```

Comment:
hard to read this line, please refactor it, move some part of logic into variable or method